src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-mount-details/cephfs-mount-details.component.html

```diff
@@ -1,37 +1,44 @@
 <cds-modal size="md"
-           [open]="open"
-           (overlaySelected)="closeModal()">
+          [open]="open" 
+          (overlaySelected)="closeModal()">
   <cds-modal-header (closeSelect)="closeModal()">
-    <h3 cdsModalHeaderHeading
-        i18n>
-      Attach commands
-    </h3>
+    <h3 cdsModalHeaderHeading i18n>Attach commands</h3>
   </cds-modal-header>
-
```

Comment:
Let's avoid pushing unnecessary code style changes as it just makes the PR harder to review and the git history harder to track